netlify_noindex_headers.txt

```diff
@@ -0,0 +1,3 @@
+# Prevent bots from indexing site
+/*
```

Comment:
Isn't this comment still unclosed?

Reply:
Oh, that's actually not an opening comment delimiter. It's specifying that the header should be for all the pages in the site. See https://www.netlify.com/docs/headers-and-basic-auth/